apps/aechannel/src/aesc_fsm.erl

```diff
@@ -2468,14 +2478,7 @@ check_closing_event_(#{ tx := SignedTx
                     {ok, proper_solo_closing, SignedTx}
             end;
         false ->
-            {ok, Hdr} = aec_chain:get_header(BlockHash),
-            Height = aec_headers:height(Hdr),
-            case aesc_channels:is_solo_closed(Ch, Height) of
-                true ->
```

Comment:
From `aesc_channels`:
```
-spec is_solo_closed(channel(), aec_blocks:height()) -> boolean().                                    
is_solo_closed(#channel{locked_until = LockedUntil} = Channel, Height) ->                             
    is_solo_closing(Channel) andalso LockedUntil =< Height.                                           
            
-spec is_solo_closing(channel()) -> boolean().
is_solo_closing(Channel) ->
    not is_active(Channel).
```
If the `aesc_channels:is_solo_closing/1` returns `false`, there is no way `aesc_channels:is_solo_closed/2` would return `true`. This was a dead code, hence removed